packages/manager/apps/pci-cold-archive/src/pages/containers/Status.components.tsx

```diff
@@ -40,7 +40,7 @@ export default function StatusComponent({
               {t(
                 'containers:pci_projects_project_storages_cold_archive_containers_status_restore_date',
                 {
-                  date: useFormattedDate(automaticDeletionAt, 'Pp'),
+                  date: useFormattedDate(automaticDeletionAt, 'P p'),
```

Comment:
I'm not sure about this change, as the docs mention `Pp`: 
https://date-fns.org/v4.1.0/docs/format